lms/views/admin.py

```diff
@@ -0,0 +1,91 @@
+from pyramid.httpexceptions import HTTPFound
+from pyramid.view import forbidden_view_config, view_config, view_defaults
+
+from lms.security import Permissions
+
+
+@view_defaults(request_method="GET")
```

Comment:
```suggestion
@view_defaults(request_method="GET", permission=Permissions.ADMIN)
```

Requires moving the forbidden view (see other comment)